### ISSUE TITLE
Doesn't work on RHEL6/CentOS6

### ISSUE THREAD

**Issue by eggsby** [deephacks/lmdbjni#45]
The liblmdbjni is compiled against GLIBC 2.14 so it won't run on EL6 which only supports 2.12 - does it really need 2.14 or could it be compiled against 2.12 so that it works on el6 as well?

**Comment by krisskross** [comment]
No, I think 2.12 should work fine. Have you tried compiling it yourself?

**Comment by navjotk** [comment]
I have the same issue as the OP above regarding GLIBC support. However the machine I will use to build has a version newer than 2.12 so I would install 2.12 only for this build. How do I point the build script to use a particular version of GLIBC and not the one in `/lib`?

**Comment by krisskross** [comment]
We're currently in the process of modernizing lmdbjni in a project called [lmdbjava](https://github.com/lmdbjava/lmdbjava). Do you mind trying it?

**Comment by navjotk** [comment]
I am only doing this to be able to run [this](https://github.com/yahoo/CaffeOnSpark) project on the cluster. I will use anything the people in that project approve.

**Comment by navjotk** [comment]
I managed to build this directly on the target machine by providing a temporary internet connection.

**Comment by krisskross** [comment]
Ok, glad to hear that!
